src/sentry/options/defaults.py

```diff
@@ -1704,6 +1704,12 @@
     default=10000,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "querybuilder.organization.project-split-by-char-optimization",
+    type=Sequence,
+    default=[],
+    flags=FLAG_AUTOMATOR_MODIFIABLE,
+)
```

Comment:
You could potentially use an options feature flag here, and just add orgs you want to opt in to the LA orgs. I assume you eventually want to roll this out everywhere, so this will enable that too.
https://develop.sentry.dev/feature-flags/#building-your-options-based-feature